tpchgen-cli/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["maturin>=1.0,<2.0"]
+build-backend = "maturin"
+
+[project]
+name = "tpchgen-cli"
+version = "1.0.0"
+description = "Python bindings for TPC-H data generator"
+requires-python = ">=3.8"
+classifiers = [
+    "Programming Language :: Rust",
+    "Programming Language :: Python :: Implementation :: CPython",
+]
+
+[project.scripts]
+tpchgen-cli = "tpchgen_cli.main:main"
+
+[tool.maturin]
+# "extension-module" tells pyo3 we want to build an extension module (skips linking against libpython.so)
+features = ["pyo3/extension-module"]
+python-source = "python"
```

Comment:
this makes maturin look at the `python/` dir, see https://www.maturin.rs/project_layout#mixed-rustpython-project